wagtail/core/tests/test_blocks.py

```diff
@@ -549,6 +549,20 @@ def validate_contains_foo(value):
         with self.assertRaises(ValidationError):
             block.clean(RichText('<p>bar</p>'))
 
+    def test_get_searchable_content(self):
+        block = blocks.RichTextBlock()
+        value = RichText(
+            '<p>Merry <a linktype="page" id="4">Christmas</a>!</p>\n'
```

Comment:
Just a small follow-up suggestion - perhaps we should include an `&amp;` in this test, to check that the `unescape` is being applied correctly...
```suggestion
            '<p>Merry <a linktype="page" id="4">Christmas</a> &amp; a happy new year!</p>\n'
```